temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using print statements for logging or debugging purposes. Use a proper logging framework to ensure better control over log levels and output formats.
```
Replace `print("hello")` with a logging statement, for example:
```python
import logging
logging.basicConfig(level=logging.INFO)
logger = logging.getLogger(__name__)
logger.info("hello")
```
```